api/src/main/resources/underlays/synpuf.yaml

```diff
@@ -108,13 +102,13 @@ datasets:
           - name: standard_concept
             dataType: STRING
           - name: vocabulary_id
-            dataType: INT64
+            dataType: STRING
```

Comment:
FYI: Some of these discrepancies may be with the OMOP version, or might be adaptations made to the AoU dataset. I know sex_at_birth is definitely an AoU extension (feedback has been given to provide clearer labeling of these fields/tables)